packages/core/src/components/dropdown/index.js

```diff
@@ -308,7 +308,21 @@ class Dropdown {
     switchClassName(this._root, ['ACTIVE', 'OPEN'], true);
   };
 
-  update = () => {
+  update = options => {
+    if (options) {
+      this.settings = {
+        ...defaults,
+        ...options
+      };
+    }
+    markupTemplates.forEach(template => {
```

Comment:
is this copy/pasted from the constructor? If so, let's make it its own function.

Reply:
done